src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -37,6 +37,8 @@
 
 logger = get_logger(__name__)
```

Comment:
btw, please make sure to copy these changes over to the /extensions/build.py as well